vocabularies/mining-survey-status.ttl

```diff
@@ -0,0 +1,63 @@
+﻿@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> ..
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix sstatus: <http://linked.data.gov.au/def/mining-survey-status/> 
+
+<http://linked.data.gov.au/def/mining-survey-status> a owl:Ontology .
+
+sstatus:conceptScheme a skos:ConceptScheme;
+  dct:created "2019-04-09T03:31:01.023Z"^^xsd:dateTime;
+  dct:modified "2019-04-09T03:34:27.949Z"^^xsd:dateTime;
+  skos:definition "The stage in the lifecycle of an exploration survey or activity (including research activities)"@en;
+  skos:prefLabel "Mining Survey or Activity Status"@en .
+
```

Comment:
Add:  
dct:creator "Geological Survey of Queensland" ;

Reply:
done